src/main/_docs/integration-guide/integration-continuous-integration.md

```diff
@@ -7,41 +7,56 @@ permalink: /:categories/continuous-integration/
 ---
 {% include base.html %}
 
-Codenvy is connected to your repo so any change made to the repo that would normally trigger a CI job will continue to trigger a CI job when the change is made in Codenvy.
+Codenvy can use [Factories]({{base}}{{site.links["factory-getting-started"]}}) with a Jenkins CI system to generate developer workspaces pre-configured with the context of the CI job. For example, a failed CI build email can be customized to include a link to two factories that open a workspace pre-configured to:
+* The branch and commit ID that broke the build for the project. 
+* The head of the master branch for the project.
```

Comment:
You are right, but 
>The head of the master branch for the project.

From this user can think that the factory configured for the webhook points to master, but in fact, accordig to the Jenkins integration flow, the factory points to developer branch. May be it would be better to write:
`The head of the branch for the project.` ?